pkg/workloads/console/acceptance/acceptance.go

```diff
@@ -0,0 +1,134 @@
+package acceptance
+
+import (
+	kitlog "github.com/go-kit/kit/log"
+	workloadsv1alpha1 "github.com/gocardless/theatre/pkg/apis/workloads/v1alpha1"
+	"github.com/gocardless/theatre/pkg/client/clientset/versioned"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+const namespace = "default"
+const consoleName = "console-0"
+
+func Run(logger kitlog.Logger) {
```

Comment:
This big method full of a bunch of assertions feels a bit off to me; would a more idiomatic Ginkgo approach be to split it up into a smaller set of `Describe` and `It`s?
This is just me thinking out loud though - maybe there's a good reason it has to be like this. Perhaps we just wait until we need to add another 'group' of acceptance assertions before thinking about this too much.

Reply:
I did this on purpose, because I think of this as a single run through of the entire flow. I think splitting it up would make it more difficult to follow.

I'm interested to see how we'd evolve this when we add more tests, but for now I do think this is the clearest way to write it.